koku/api/iam/view/customer.py

```diff
@@ -16,17 +16,39 @@
 #
 
 """View for Customers."""
+import logging
 
-from rest_framework import mixins, viewsets
+from django.contrib.auth.models import Group
+from django.db import DatabaseError, transaction
+from django.utils.encoding import force_text
+from rest_framework import mixins, status, viewsets
 from rest_framework.authentication import (SessionAuthentication,
                                            TokenAuthentication)
+from rest_framework.exceptions import APIException
 from rest_framework.permissions import IsAdminUser
 
 import api.iam.models as model
 import api.iam.serializers as serializers
 
 
+logging.basicConfig()
+LOG = logging.getLogger(__name__)
+LOG.setLevel(logging.INFO)
```

Comment:
You can create a logger but the level and initial setup should be handled by the Django settings.